cfme/tests/cloud/test_cloud_events.py

```diff
@@ -90,7 +90,7 @@ def test_vm_capture(appliance, request, provider, register_event):
 
     def cmp_function(_, y):
         # In 5.9 version `y` is a dict, not a yaml stream.
-        data = yaml.load(y) if appliance.version < '5.9' else y
+        data = yaml.safe_load(y) if appliance.version < '5.9' else y
```

Comment:
**Optional** don't think we need the < 5.9 here, but probably out of scope for this PR.

Reply:
Yeah, just gonna leave this one in for now.